test/revision

```diff
@@ -1 +1 @@
-11453
+13645
```

Comment:
This only affects the binaries that get downloaded from the waterfall. Unfortunately (and I actually forget why ATM) the tests in `tests/torture-s` are just checked directly in. So you have to just download the torture-s archive from the waterfall (e.g. https://storage.googleapis.com/wasm-llvm/builds/linux/13639/wasm-torture-s-13639.tbz2), replace the existing `tests/torture-s` directory with that, and commit that.